client/tests/functional/details/CurlConnectionTests.cpp

```diff
@@ -95,14 +95,11 @@ TEST("Testing CurlConnection()")
     handler.SetLoggingCallback(LogCallbackToTest);
     CurlConnectionManager connectionManager(handler);
     auto connection = connectionManager.MakeConnection({});
+    SFSUrlBuilder urlBuilder(SFSCustomUrl(server.GetBaseUrl()), c_instanceId, c_namespace, handler);
 
     SECTION("Testing CurlConnection::Get()")
     {
-        const std::string url = SFSUrlComponents::GetSpecificVersionUrl(server.GetBaseUrl(),
-                                                                        c_instanceId,
-                                                                        c_namespace,
-                                                                        c_productName,
-                                                                        c_version);
+        const std::string url = urlBuilder.GetSpecificVersionUrl(c_productName, c_version);

```

Comment:
nit: when to use = `const std::string url = ...` vs. const `std::string url{ ... }`?

Reply:
it's was a question of style, not performance.